src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -541,7 +593,7 @@ if (typeof Sfjs === 'undefined') {
                         }
                     },
                     function(xhr) {
-                        if (xhr.status !== 0) {
+                        if (xhr.status !== 0 && !options.stop) {
```

Comment:
IMHO the check on `options.stop` should also be performed inside the `setTimeout` closure, to prevent a new request to be performed.

ie:
- `request` is called
- the xhr failed and `setTimeout` is scheduled for 1 second
- 100ms later, the user click on `cancel`
- 900ms latter the `setTimeout` wakeup and call `request` again